pkg/cloudprovider/providers/aws/aws.go

```diff
@@ -3323,13 +3323,22 @@ func (c *Cloud) EnsureLoadBalancer(ctx context.Context, clusterName string, apiS
 			instanceIDs = append(instanceIDs, string(id))
 		}
 
+		loadBalancerAttributes := map[string]string{}
+
+		// Determine if cross zone load balancing enabled/disabled has been specified
+		crossZoneLoadBalancingEnabledAnnotation := annotations[ServiceAnnotationLoadBalancerCrossZoneLoadBalancingEnabled]
+		if crossZoneLoadBalancingEnabledAnnotation != "" {
+			loadBalancerAttributes["load_balancing.cross_zone.enabled"] = crossZoneLoadBalancingEnabledAnnotation
```

Comment:
This value should be `"true"` not whatever string value the user entered.

Reply:
The reasoning behind using the value here, was to make setting it explicitly to `false` work. I know that you could also delete the annotation to revert it to `false`, but that seemed not really obvious. wdyt?

Reply:
Yea, actually `true` or `false` could be valid values. Can you make sure you call `strconv.ParseBool()` like the check for access logs does to make sure the input is valid? https://github.com/kubernetes/kubernetes/blob/master/pkg/cloudprovider/providers/aws/aws.go#L3415-L3425

Reply:
Fixed